3-networks/modules/standard_shared_vpc/main.tf

```diff
@@ -18,36 +18,79 @@
   Shared VPC configuration
  *****************************************/
 
+locals {
+  vpc_type                = "shared"
+  vpc_label               = (var.vpc_label != "" ? var.vpc_label : "private")
```

Comment:
is this required, should we just not use the naming in the whitepaper? We could also remove the variable then